parsing/comments.ml

```diff
@@ -0,0 +1,167 @@
+(***********************************************************************)
+(*                                                                     *)
+(*                                OCaml                                *)
+(*                                                                     *)
+(*            Xavier Leroy, projet Cristal, INRIA Rocquencourt         *)
+(*                                                                     *)
+(*  Copyright 1996 Institut National de Recherche en Informatique et   *)
```

Comment:
Please be a bit more careful with the copyright headers. I know it's annoying to have to pay attention to them, but it's not that hard.